src/edit/command.js

```diff
@@ -102,7 +102,7 @@ export class CommandSet {
     this.op = op
   }
 
-  // :: (union<Object<CommandSpec>, string>, ?(string, CommandSpec) → bool) → CommandSet
+  // :: (union<Object<CommandSpec>, "schema">, ?(string, CommandSpec) → bool) → CommandSet
```

Comment:
Not sure I got that right. Do you propose to just parse `"[^"]+"` as another valid type and output it as is? If that's the case, I think that would look ok in the reference and be understandable.